src/Helper/VersionFromUri.php

```diff
@@ -29,10 +29,10 @@ public function __construct(Request $request)
     public function getVersion(): string
     {
         $uri = $this->request->getPathInfo();
-        if (! preg_match('/\/v([0-9.]+)\//i', $uri, $version)) {
```

Comment:
Thanks!

Would you like to add your situation as a test https://github.com/reindert-vetter/api-version-control/blob/master/test/Unit/VersionParser/VersionFromUriTest.php#L31-L54? Feel free to add other tests if you can better reproduce your situation.

I don't think we should see `/v9motor` as a version. So I think you should also take into account that you don't want a string character after the version (except for a `/`).

Of course you can always copy VersionFromUri.php and define your own version in the config file.

Reply:
Added a test to test this:
```
            [
                '/test/v3.0',
                '3.0',
            ],
```

However, then the following falls:
```php
            [
                '/v/v3.0suffix/test',
                '0',
            ],
            [
                '/v3motor',
                '0.0',
            ],
```
I can't really get it to work.

Reply:
You are right! I will look for a way to to solve it and also add some tests when I get a chance!